_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any `img` element that is [visible][] except if the `src` [attribute value][] references an SVG document.
+
+## Expectation
+
+The image resource referenced by the `src` [attribute value][] of each target element either does not contain text expressing anything in a [human language][] or it is [essential][] that the text is rendered with that specific presentation.
```

Comment:
This should actually look at each element in the source set of the image.
Check https://act-rules.github.io/rules/9eb3f6 where I did that. It's fairly easy to do and will make the rule slightly more powerful. May need to change Applicability accordingly.

Reply:
Good suggestion. And it also makes sense to include image buttons.